### PR TITLE
Added console & minio domains selector to add tenant wizard

### PR DESCRIPTION
**Pull request by bexsoft**
## What does this do?

Added console & minio domains selector to add tenant wizard

## How does it look?

Screen Shot 2022-04-18 at 18 50 41
Screen Shot 2022-04-18 at 18 50 16
Screen Shot 2022-04-18 at 18 50 10
Screen Shot 2022-04-18 at 18 49 57

Signed-off-by: Benjamin Perez <benjamin@bexsoft.net>

### CODE REVIEW

**Review by dvaldivia — CHANGES_REQUESTED** [review]
There's no need for schema in the URLs, in fact the validation should tell them not to enter the schema since these domains go into the `MINIO_DOMAIN` env var. but it does raise the question about the public domain.

Also minio domains are not getting populated

**Comment by bexsoft** [comment]
As per conversation, I will add validation to include protocol & ports.

Also MinIO domain should not allow subpaths

**Comment by bexsoft** [comment]
> There's no need for schema in the URLs, in fact the validation should tell them not to enter the schema since these domains go into the `MINIO_DOMAIN` env var. but it does raise the question about the public domain.
> 
> Also minio domains are not getting populated 

Added validation to input boxes & fixed issue with one minio domain @dvaldivia

**Review by dvaldivia — APPROVED** [review]

**Review by Alevsk — APPROVED** [review]

File: portal-ui/src/screens/Console/Tenants/AddTenant/Steps/Configure.tsx [hunk 1]
```diff
@@ -79,11 +92,25 @@ const styles = (theme: Theme) =>
...
+    overlayAction: {
+      marginLeft: 10,
+      display: "flex",
+      alignItems: "center",
+      "& svg": {
+        width: 15,
+        height: 15,
+      },
+      "& button": {
+        background: "#EAEAEA",
```
Comment (Alevsk) [inline portal-ui/src/screens/Console/Tenants/AddTenant/Steps/Configure.tsx:111]: EAEAEAEAE XD

**Pull request merged**